rest-api/config.py

```diff
@@ -51,6 +51,9 @@
 
 DAYS_TO_DELETE_KEYS = "days_to_delete_keys"
 
+# service accounts exception from key deletion
+SERVICE_ACCOUNT_KEYS = "service_account_keys"
```

Comment:
I would rename the constant name and value to "service_accounts_with_long_lived_keys" or something, to make it clear what this means.